packages/salesforcedx-vscode-apex/tsconfig.json

```diff
@@ -1,22 +1,16 @@
 {
-    "compilerOptions": {
-        "module": "commonjs",
-        "target": "es6",
-        "lib": [
-            "es6"
-        ],
-        "sourceMap": true,
-        "declaration": true,
-        "moduleResolution": "node",
-        "noImplicitAny": true,
-        "rootDir": ".",
-        "outDir": "out",
-        "preserveConstEnums": true,
-        "strict": true
-    },
-    "exclude": [
-        "node_modules",
-        ".vscode-test",
-        "out"
-    ]
-}
\ No newline at end of file
+  "compilerOptions": {
```

Comment:
Was this just a formatting or did we change anything here?

Reply:
Got it. And this is formatted using prettier?

Reply:
Yeah, I ran node scripts/reformatwithprettier.js so it should be formatted with that